examples/ibm-catalog/standard-solution/main.tf

```diff
@@ -26,12 +26,18 @@ provider "ibm" {
   ibmcloud_api_key = var.ibmcloud_api_key != null ? var.ibmcloud_api_key : null
 }
 
+locals {
+  location = regex("us-south|us-east|eu-de|eu-gb", var.prerequisite_workspace_id)
+}
```

Comment:
what happens when schematics add support for more regions? We need to maintain the list here. Can we future proof this and return the string before the first dot?
@argeiger had proposed a regex that could work for this

Reply:
let me check. Just back from vacation. Is it on slack channel ?